oak/server/logging_node.h

```diff
@@ -32,6 +32,7 @@ class LoggingNode final : public NodeThread {
   explicit LoggingNode(BaseRuntime* runtime, const std::string& name) : NodeThread(runtime, name) {}
 
  private:
+  static const std::string log_levels[5];
```

Comment:
Not sure why this needs to be a member of this class, as opposed to just a constant in an unnamed namespace?

Reply:
Fixed. Found that protobuf compiler generates a function that returns the enum name corresponding to a numeric value.